autopilot/agent.go

```diff
@@ -256,6 +261,13 @@ type chanPendingOpenUpdate struct{}
 // a previous channel open failed, and that it might be possible to try again.
 type chanOpenFailureUpdate struct{}
 
+// heuristicUpdate is an update sent when one of the autopilot heuristics has
+// changed, and prompts the agent to make a new attempt at opening more
+// channels.
+type heuristicUpdate struct {
+	heuristic AttachmentHeuristic
+}
```

Comment:
Are we expecting more fields to be added to this struct other than a `AttachmentHeuristic`? 
I'm not clear on the purpose of this struct if it just wraps a single heuristic?

Reply:
Good question! It could definitely work without the extra struct, it's just there to mimic the other event types we have here, so felt like we should use the same pattern.

Reply:
Ok cool, LGTM otherwise 😄